Makefile

```diff
@@ -115,8 +115,13 @@ swagger: swagger/docs.go
 
 .PHONY: update-licenses
 update-licenses:
-	go get github.com/google/go-licenses
-	$(shell echo "Module | License URL | License" > THIRD-PARTY-LICENSES.md; echo "---|---|---" >> THIRD-PARTY-LICENSES.md; for i in $$(cat go.sum  | awk '{print $$1}'); do l=$$(go-licenses csv $$i 2>/dev/null); if [ $$? -ne 0 ]; then continue; fi; echo $$l | tr \, \| | tr ' ' '\n'; done | sort -u >> THIRD-PARTY-LICENSES.md)
+	go install github.com/google/go-licenses@latest
+	$(shell echo "Module | License URL | License" > THIRD-PARTY-LICENSES.md; echo "---|---|---" >> THIRD-PARTY-LICENSES.md; for i in $$(go list -m all  | awk '{print $$1}'); do l=$$(go-licenses csv $$i 2>/dev/null); if [ $$? -ne 0 ]; then continue; fi; echo $$l | tr \, \| | tr ' ' '\n'; done | sort -u >> THIRD-PARTY-LICENSES.md)
+
+.PHONY: check-licenses
+check-licenses:
+	go install github.com/google/go-licenses@latest
+	$(shell for i in $$(go list -m all  | awk '{print $$1}'); do l=$$(go-licenses check $$i 2>/dev/null); if [ $$? -ne 0 ]; then continue; fi; done)
```

Comment:
How this will fail in case a license is in forbidden list? (If I read it right it will never fail due to continue). Also maybe it makes sense not to redirect errors to /dev/null and display them.

Also how about build flags(extended,minimal)?  What happens if not specified like it is currently in the PR?

Reply:
PR updated.

```
Checking github.com/briandowns/spinner ... OK
Checking github.com/chartmuseum/auth ... OK
Checking github.com/containers/common ... UNKNOWN
Checking github.com/containers/image/v5 ... OK
Checking github.com/didip/tollbooth/v6 ... OK
Checking github.com/docker/distribution ... OK
Checking github.com/dustin/go-humanize ... OK
Checking github.com/fsnotify/fsnotify ... OK
Checking github.com/getlantern/deepcopy ... OK
Checking github.com/go-ldap/ldap/v3 ... OK
Checking github.com/gofrs/uuid ... OK
Checking github.com/google/go-containerregistry ... UNKNOWN
Checking github.com/google/uuid ... OK
Checking github.com/gorilla/handlers ... OK